select2-parent/select2/src/main/java/org/wicketstuff/select2/Settings.java

```diff
@@ -58,7 +57,7 @@
 
 	private AjaxSettings ajax;
 	private String data;
-	private String tags;
+	private Boolean tags;
```

Comment:
Actually all Boolean options are JS Objects
so I believe it is safe to change all Boolean to be boolean.

I'll change it right now